addons/common/XEH_postInit.sqf

```diff
@@ -446,4 +446,9 @@ GVAR(deviceKeyCurrentIndex) = -1;
 {false},
 [0xC7, [true, false, false]], false] call CBA_fnc_addKeybind;  //SHIFT + Home Key
 
+
+// init persistent laser
+if (GVAR(persistentLaserEnabled)) then {true call FUNC(switchPersistentLaser)};
```

Comment:
I don't think we need this as the function should be run when cba settings are ready

Reply:
will check this

Reply:
it's true, didn't know that, fixed both comments